igcollect/ping.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+
+"""igcollect - Latency collection
+Copyright (c) 2020 InnoGames GmbH
+"""
+
+# This script is used to collect ping data to evaluate availability and latency
+# of our network and make it easy to create a Graphite Dashboard
+#
+# Please note that the package fping has to be installed on your Linux server!
+
+
+import argparse
+import subprocess
+import time
+import socket
+
+from multiprocessing import Pool
+from os import cpu_count
+from statistics import stdev
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Ping collector script for Graphite ')
+    parser.add_argument('hosts', nargs='+',
+                        help='the hosts to ping')
+    parser.add_argument('-c', '--count', type=int, default=20,
+                        choices=range(1,60),
+                        help='how many times the script will try to ping '
+                             '(number has to be bigger '
+                             'than 1 and smaller than 60) (default= 20)')
+    parser.add_argument('-p', '--prefix',
+                        default='servers.{}.system.ping'.format(
+                            socket.gethostname().replace('.', '_')),
+                        help='the path to the value in Graphite '
+                             '(default= servers.(hostname).system.ping)')
+    parser.add_argument('-t', '--timeout', type=int, default=500,
+                        help='Initial target timeout in milliseconds. '
+                             'the amount of time that program '
+                             'waits for a response (default= 500)')
+    parser.add_argument('-d', '--delay', type=int, default=1800,
+                        choices=range(0,5000),
+                        help='time to wait between a series of pings in '
+                             'milliseconds (default= 1800 for 20 pings)')
```

Comment:
You're duplicating defaults everywhere. Use https://docs.python.org/3/library/argparse.html#argparse.ArgumentDefaultsHelpFormatter instead.